lib/spack/spack/test/conftest.py

```diff
@@ -388,6 +388,7 @@ def install_mockery(tmpdir, config, mock_packages):
     with spack.config.override('config:checksum', False):
         yield
 
+    tmpdir.join('opt').remove()
```

Comment:
why is this extra line here?

Reply:
the fixture didn't properly clean up after itself and triggered cleanup checks in some of the other fixtures.